pkg/api/grpcServer.go

```diff
@@ -61,9 +61,119 @@ type (
 )
 
 var (
-	// TODO: need full list
+	// global word list as defined in https://github.com/istio/api/blob/master/mixer/v1/global_dictionary.yaml
```

Comment:
add comment to state which version of the file. now it is the first version.
we can add a version comment in the global dictionary file next time we update it.